blob.py

```diff
@@ -119,9 +120,19 @@ def download(context, backend, handle, output, format):
     type=click.Choice(["GENERIC", "RANDOM"], case_sensitive=False),
 )
 @click.option("--tempo", default=1.0, type=float)
+@click.option(
+    "--tracks",
+    default="AUTO",
+    type=str,
+)
```

Comment:
Click can automatically parse and validate argument lists for us.

```suggestion
@click.option(
    "--tracks",
    default=(0 1 -2 -1),
    type=int,
    nargs=4,
    help="Indexes for soprano, alto, tenor and bass.",
    show_default=True
)
```

The invocation would look like this:

```bash
python blob.py ··· --tracks 1 2 -2 -1
```